config/engineering/metatag.metatag_defaults.node__spotlight.yml

```diff
@@ -0,0 +1,24 @@
+uuid: 3358306e-5082-4513-acc9-dcace84c4215
+langcode: en
+status: true
+dependencies: {  }
+id: node__spotlight
+label: 'Content: Spotlight'
+tags:
+  description: '[node:body|node:su_people_spotlight_quote]'
+  keywords: '[node:su_spotlight_affiliation]'
+  og_description: '[node:body|node:su_people_spotlight_quote]'
+  og_image: '[node:su_spotlight_feature_image:entity:field_media_image:card_1192x596]'
+  og_image_alt: '[node:su_spotlight_feature_image:entity:field_media_image:alt|node:title]'
+  og_image_url: '[node:su_spotlight_feature_image:entity:field_media_image:card_1192x596]'
+  og_site_name: '[site:name]'
+  og_title: '[node:title]'
+  og_type: website
+  og_url: '[current-page:url:absolute]'
+  profile_first_name: '[node:su_spotlight_person_title]'
+  title: '[node:title] | [site:name]'
```

Comment:
You only need to specify metatags that you want to be different than the [default set](https://github.com/SU-SOE/engineering_profile/blob/11.x/config/sync/metatag.metatag_defaults.node.yml). So this file can be trimmed down to only the things that are spotlight specific.